components/server/src/jobs/workspace-gc.ts

```diff
@@ -187,6 +243,41 @@ export class WorkspaceGarbageCollector implements Job {
         }
     }
 
+    private async deleteEligiblePrebuilds() {
```

Comment:
Why does this need to be a separate function, now that we can unify over the `deletionEligibilityDate`? For more precise logging? :thinking:

Reply:
Because deleting a prebuild is a different logic, I would need to check the type first thing and then delegate to two different functions again.

Reply:
> I would need to check the type first thing and then delegate to two different functions again.

Yep. But realized now that we only select `id, ownerId`. :bulb: And the additional context for logging is nice as well. :+1: